code/modules/clothing/under/accessories/accessory_vr.dm

```diff
@@ -31,6 +64,39 @@
 	icon_override = 'icons/mob/ties_vr.dmi'
 	icon_state = "collar_blk"
 	var/writtenon = 0
+	var/icon_previous_override //yw addition
```

Comment:
likewise, Vorestation Edit should be used here.

Reply:
Was just noting where I got this from as it's already a _VR file